e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -299,4 +309,12 @@ private void assertThatNotDocumentsWereAdded(Document gpcAccessDocument) {
         var documentList = gpcAccessDocument.get("documents", Collections.emptyList());
         assertThat(documentList).isNotEmpty();
     }
+
+    private static String getEnvVar(String name, String defaultValue) {
```

Comment:
that is small repo so potentially not a candidate for much of reuse ;) so ok to be here probably.